src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -141,3 +141,68 @@ RppStatus rppt_crop_host(RppPtr_t srcPtr,
 
     return RPP_SUCCESS;
 }
+
+/******************** crop mirror normalize ********************/
+
+RppStatus
+rppt_crop_mirror_normalize_gpu(RppPtr_t srcPtr,
```

Comment:
Format the rppt_ function name right next to the RppStatus like in crop or other functions. Please change the same for cmn host api too.